PULL_REQUEST_TEMPLATE.md

```diff
@@ -15,6 +15,31 @@ Please delete options that are not relevant.
 - [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
 - [ ] Documentation updated
 
+## Conventional Commits guidelines
+
+- [ ] I made sure the PR title follows the 
+https://www.conventionalcommits.org/en/v1.0.0/
+
+
+Changes to workflow inputs (sample table and/or configs)
+* major (add **BREAKING CHANGE:** in the beginning of the PR title)
```

Comment:
Would every PR increase the version (major, minor or patch)?

Reply:
This has to do with changelog. There are essentially 2 happening in this PR: 
1. automatic annotation of the release and the changes happening
2. changelog, which should be updated each time we push a PR